test/test_tensordict.py

```diff
@@ -296,6 +296,24 @@ def test_permute(device):
     assert torch.sum(t["a"]) == torch.Tensor([0])
 
 
+@pytest.mark.parametrize("device", get_available_devices())
+def test_permute_applied_twice(device):
```

Comment:
It would be cool to move this to `TestTensorDict`
Here's what it will look like

```python
...
class TestTensorDict:
    ...
    def test_permute_applied_twice(self, td_name, device):
        torch.manual_seed(0)
        tensordict = getattr(self, td_name)(device)
        for _ in range(10):
             p =  torch.randperm(4)
             inv_p = p.argsort()
             other_p = inv_p
             while (other_p == inv_p).all():
                 other_p = torch.randperm(4)
             assert tensordict.permute(*p).permute(*inv_p) is tensordict
             assert tensordict.permute(*p).permute(*other_p) is not tensordict
             
```

Reply:
Hi @vmoens , there is no class in the test files, we are using `pytest` and it doesn't work like that

Reply:
Actually it does :-)
Have a look at [this](https://github.com/facebookresearch/rl/blob/850381115c074bba89da23a6a66d7485bbc366af/test/test_tensordict.py#L519)
It would be great if we could fit this test in that. You can pretty much copy paste what i have written above

Reply:
Sorry for the confusion @vmoens, I have added your method as you requested